File: .sdk/tm/php/utility/PreparePath.php
<?php
declare(strict_types=1);

// SeqbenchMcp SDK utility: prepare_path

class SeqbenchMcpPreparePath
{
    public static function call(SeqbenchMcpContext $ctx): string
    {
        $point = $ctx->point;
        $parts = [];
        if ($point) {
            $p = \Voxgig\Struct\Struct::getprop($point, 'parts');
            if (is_array($p)) {
                $parts = $p;
            }
        }
        return \Voxgig\Struct\Struct::join($parts, '/', true);
    }
}

File: go/utility/prepare_path.go
package utility

import (
	vs "github.com/voxgig-sdk/seqbench-mcp-sdk/go/utility/struct"

	"github.com/voxgig-sdk/seqbench-mcp-sdk/go/core"
)

func preparePathUtil(ctx *core.Context) string {
	point := ctx.Point

	var parts []any
	if p := vs.GetProp(point, "parts"); p != nil {
		if pl, ok := p.([]any); ok {
			parts = pl
		}
	}

	path := vs.Join(parts, "/", true)
	return path
}
